README.md

```diff
@@ -20,7 +20,7 @@ You will need:
 Install dependencies:
 
 ```
-$ yarn install
+$ yarn install --production=false
```

Comment:
Yeah, it's necessary to pass this to Travis. Travis has `NODE_ENV` set to `production` when it runs `yarn install`, so `devDependencies` aren't installed.

However developers (who this README is aimed at) will not have `NODE_ENV` set to `production` on their dev machines, so it is not necessary for them to set the `--production=false` flag to ensure `devDependencies` will be installed.

Does that make sense?